modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm

```diff
@@ -282,7 +302,7 @@
 				hit_things |= slapped
 		if(!spinning)
 			break
-		sleep(1.25)
+		sleep(0.05 SECONDS)
```

Comment:
Why does there need to be a sleep in here? Use timers and callbacks instead, please.